app/src/main/java/com/ustwo/boilerplate/application/Application.java

```diff
@@ -0,0 +1,16 @@
+package com.ustwo.boilerplate.application;
+
+public class Application extends android.app.Application {
```

Comment:
Would prefer naming this class BoilerplateApplication rather than having two 'Application' classes in the project.